docker/build_jar.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+# Get the last commit hash of the model directory and set it as an environment variable
+GIT_MODEL_HASH=$(git log -n 1 --pretty=format:%h)
+
+docker run --rm -v $PWD:/working -v jar-cache:/root/.ivy -v jar-cache:/root/.ivy2 broadinstitute/scala-baseimage:jdk11-2.13.5-1.4.7 /working/docker/clean_install.sh /working
```

Comment:
Note Sam JDK I think was just updated to java 17, so we might want to use a new base image now.

Reply:
is there a way we can dynamically pull the "correct" java version in the sam repo?

Reply:
Updated the base image to `hseeberger/scala-sbt:eclipse-temurin-17.0.2_1.6.2_2.13.8` so this should be OK for now.